scripts/translations.py

```diff
@@ -0,0 +1,98 @@
+import argparse
+from upload import upload_task
+
+parser = argparse.ArgumentParser()
+parser.add_argument('-f', '--force', dest='force', action="store_true", default=False)
+args = parser.parse_args()
+
+languages = [
+    ('Simplified Chinese'),
+    ('German'),
+    ('Russian'),
+    ('Italian'),
+    ('French'),
+    ('Turkish'),
+    ('Portuguese'),
+    ('Arabic'),
+    ('Hindi'),
+]
+
+description = """Zulip is a tool used by many teams daily. In order to make it more approachable
+to users, there is an ongoing translation effort. [Read more about translating
+Zulip](https://zulip.readthedocs.io/en/latest/translating.html).
+
+This group of tasks is about creating common translation style guides, to give
+guidance on how Zulip should be translated into specific languages. Currently
+there are guides for [Polish](https://zulip.readthedocs.io/en/latest/polish.html)
+and [Spanish](https://zulip.readthedocs.io/en/latest/spanish.html).
+
+"""
+
+description_ABC = description + "For this task, do **Task Type %(type)s** for " + \
+                  "**%(language)s**"
+
+for language in languages:
+    #Task Type A
+    upload_task(
+        # https://developers.google.com/open-source/gci/resources/downloads/TaskAPISpec.pdf
+        name = 'Research web application translations into %s' % (language),
```

Comment:
A lot of the tasks I wrote don't do this, but I would put the %s earlier in the name, so that it doesn't get cut-off by the narrow "name" column in google's task display. So something like "Research %s web application translations."